packages/ruleset-bundler/README.md

```diff
@@ -13,3 +11,24 @@
 - treeshake - whether to enable tree shaking. False by default.
 
 **Bolded** options are required.
+
+## Loading YAML/JSON Ruleset
+
+It is assumed you have [@stoplight/spectral-ruleset-migrator](https://www.npmjs.com/package/@stoplight/spectral-ruleset-migrator) installed.
```

Comment:
Any reason to not have this as a dependency if it's always required? Env issue, dedupe issue, something else?